### PR TITLE
#60: Conversation and Messages model

### PR DESCRIPTION
**Pull request by evankp**
- [X] Create Conversation model (I know ticket is for only one-to-one messaging, but I feel this model could be expanded easily, as users array seems to be a good way to go to me)

- [X] Create a single message model

### CODE REVIEW

**Comment by evankp** [comment]
Please let me know how this could be improved if it can since it is my first time creating a model

**Review by DevDelvo — APPROVED** [review]
Looks solid.

File: server/models/Message.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+const mongoose = require('mongoose');
```
Comment (tsyaeger) [inline server/models/Message.js]: Just wanted to throw out options for seen or received bool fields.

File: server/models/Conversation.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+const { ObjectId } = Schema;
+
+const ConversationSchema = new Schema({
```
Comment (tsyaeger) [inline server/models/Conversation.js]: Date fields for both of these models (like Kevin used in Project) would be helpful.

**Comment by evankp** [comment]
@DevDelvo @tsyaeger Don't think I need to re-request review, but I'm going to merge this tomorrow morning anyhow, seeing as there will most likely not be any conflicts 😆

**Pull request merged**